src/settings/StaffSlips/StaffSlipForm.js

```diff
@@ -68,7 +67,7 @@ class StaffSlipForm extends React.Component {
     const { pristine, submitting, initialValues } = this.props;
     const edit = initialValues && initialValues.id;
     const saveLabel = edit
-      ? <FormattedMessage id="ui-circulation.settings.staffSlips.saveAndClose" />
+      ? <FormattedMessage id="stripes-core.button.save" />
```

Comment:
Why do we need to change the label?

Reply:
It is a requirement of the story https://issues.folio.org/browse/UICIRC-316. 4-th point